xml/deploy_cephadm.xml

```diff
@@ -791,6 +791,20 @@ o- time_server ................................................ [enabled]
      configuration:
     </para>
 <screen>&prompt.smaster;ceph-salt config /system_update/reboot disable</screen>
+   </sect3>
+   <sect3 xml:id="deploy-cephadm-enable-network">
+     <title>Configure The Cluster Network</title>
+     <para>
+       Optionally, if you are running a separate cluster you may need to enable
```

Comment:
`if you are running a separate cluster` separate cluster network ? or separate osd network ?

Reply:
Should be cluster network, true.